tests/integration/security/authorization_test.go

```diff
@@ -544,38 +545,123 @@ func TestAuthorization_EgressGateway(t *testing.T) {
 			defer ctx.Config().DeleteYAMLOrFail(t, "", policies...)
 
 			cases := []struct {
-				path string
-				code string
-				body string
+				name  string
+				path  string
+				code  string
+				body  string
+				host  string
+				from  echo.Workload
+				token string
 			}{
 				{
+					name: "allow path to company.com",
```

Comment:
Isn't this (and the following one) also over mTLS now according to the virtual service? (port 80 is routed to 443), this is fine but the name should be updated like other test cases.

Reply:
no.. if you look at the virtual service, company.com goes to port 80 of gateway. it is not rerouted